tests/networking/tests/networking_multus_links.go

```diff
@@ -0,0 +1,257 @@
+		Expect(err).ToNot(HaveOccurred())
+	})
+
+	BeforeEach(func() {
+
+		By("Clean namespace before each test")
+		err := namespaces.Clean(netparameters.TestNetworkingNameSpace, globalhelper.APIClient)
+		Expect(err).ToNot(HaveOccurred())
+
+		By("Remove reports from report directory")
+		err = globalhelper.RemoveContentsFromReportDir()
+		Expect(err).ToNot(HaveOccurred())
+
+		if len(multusInterfaces) < 1 {
+			Skip("There is no enough Multus interfaces available")
+		}
+	})
+
+	// 48328
+	It("custom deployment 3 pods, 1 NAD, connectivity via Multus secondary interface",
+		func() {
+
+			By("Define and create Network-attachment-definition")
+			err := nethelper.DefineAndCreateNadOnCluster(
+				netparameters.TestNadNameA, multusInterfaces[0], netparameters.TestIPamIPNetworkA)
+			Expect(err).ToNot(HaveOccurred())
+
+			By("Define deployment and create it on cluster")
+			err = nethelper.DefineAndCreateDeploymentWithMultusOnCluster(
+				netparameters.TestDeploymentAName, netparameters.TestNadNameA, 3)
+			Expect(err).ToNot(HaveOccurred())
+
+			By("Start tests")
+			err = globalhelper.LaunchTests(
+				[]string{netparameters.NetworkingTestSuiteName},
+				netparameters.TestCaseMultusSkipRegEx,
+			)
+			Expect(err).ToNot(HaveOccurred())
+
+			By("Verify test case status in Junit and Claim reports")
+			err = globalhelper.ValidateIfReportsAreValid(
+				netparameters.TestCaseMultusConnectivityName,
+				globalparameters.TestCasePassed)
+			Expect(err).ToNot(HaveOccurred())
+		})
+
+	// 48330
+	It("2 custom deployments 3 pods, 1 NAD, connectivity via Multus secondary interface",
+		func() {
+
+			By("Define and create Network-attachment-definition")
+			err := nethelper.DefineAndCreateNadOnCluster(
+				netparameters.TestNadNameA, multusInterfaces[0], netparameters.TestIPamIPNetworkA)
+			Expect(err).ToNot(HaveOccurred())
+
+			By("Define first deployment and create it on cluster")
+			err = nethelper.DefineAndCreateDeploymentWithMultusOnCluster(
+				netparameters.TestDeploymentAName, netparameters.TestNadNameA, 3)
+			Expect(err).ToNot(HaveOccurred())
+
+			By("Define second deployment and create it on cluster")
+			err = nethelper.DefineAndCreateDeploymentWithMultusOnCluster(
+				netparameters.TestDeploymentBName, netparameters.TestNadNameA, 3)
+			Expect(err).ToNot(HaveOccurred())
+
+			By("Start tests")
+			err = globalhelper.LaunchTests(
+				[]string{netparameters.NetworkingTestSuiteName},
+				netparameters.TestCaseMultusSkipRegEx,
+			)
+			Expect(err).ToNot(HaveOccurred())
+
+			By("Verify test case status in Junit and Claim reports")
+			err = globalhelper.ValidateIfReportsAreValid(
+				netparameters.TestCaseMultusConnectivityName,
+				globalparameters.TestCasePassed)
+			Expect(err).ToNot(HaveOccurred())
+			time.Sleep(30 * time.Second)
+		})
+
+	// 48331
+	It("custom deployment and daemonset 3 pods, 2 NADs, connectivity via Multus secondary interfaces",
```

Comment:
Maybe we could also add a scenario with 2 NADs in the same deployment? Did not cat it during the testcase review

Reply:
Sure, I can do that in the next PR